### ISSUE TITLE
Review loss of axioms

### ISSUE THREAD

**Issue by matentzn** [obophenotype/cell-ontology#573]
- SubClassOf(<http://purl.obolibrary.org/obo/CL_0000108> <http://purl.obolibrary.org/obo/CL_0000381>)
- SubClassOf(<http://purl.obolibrary.org/obo/CL_0000125> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002202> <http://purl.obolibrary.org/obo/CL_0000030>))
- SubClassOf(<http://purl.obolibrary.org/obo/CL_0008016> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002202> <http://purl.obolibrary.org/obo/CL_0008012>))

(Redundancy stripping?)

**Comment by paolaroncaglia** [comment]
@shawntanzk @gouttegd  I'm reviewing a previous note by @matentzn investigating loss of axioms. The top one is about neurons:

> SubClassOf(http://purl.obolibrary.org/obo/CL_0000108 http://purl.obolibrary.org/obo/CL_0000381)

Should 'cholinergic neuron' be a subclass of 'neurosecretory neuron'? The latter is asserted as being an efferent neuron: is that correct across taxa? ('neurosecretory neuron' has an FBbt xref.) Are all cholinergic neurons efferent? I'm not convinced that we should add that subclass.

**Comment by paolaroncaglia** [comment]
@dosumis 
The original request in this ticket was a note from Nico to review loss of 3 axioms. The last one pertains to 2 terms that you created. The lost axiom is 
'activated skeletal muscle satellite cell' develops from 'quiescent skeletal muscle satellite cell'
Their common parent 'skeletal muscle satellite cell' is textually defined as "...These cells are mostly quiescent, but upon activation they divide to produce cells that generate new muscle fibers." and 'activated skeletal muscle satellite cell''s def says "A skeletal muscle cell that has become mitotically active - typically following muscle damage.". Therefore, I believe that the lack/loss of that axiom is correct. Could you confirm please?

**Comment by shawntanzk** [comment]
> SubClassOf(http://purl.obolibrary.org/obo/CL_0000108 http://purl.obolibrary.org/obo/CL_0000381)

I think this might come down to if acetylcholine is secreted as a hormone by cholinergic neurons. I had to google this cause I forgot my undergrad on which neurotransmitter is a hormone too >.< and like everything, the answer isn't super simple. I THINK that the removal of this axiom is right as ACh from the neuronal system doesn't pass the blood brain barrier (from wiki), and this paper where ACh is noted as a hormone (https://academic.oup.com/endo/article/145/5/2498/2878537) calls it nonneuronal cholinergic signaling. 

tl;dr: removal of this axiom is probably right.

**Comment by shawntanzk** [comment]
> glial cell is taxon-agnostic, while glioblast has subclasses specific to invertebrates vs. vertebrates. Is it true/known for all taxa that glial cells develop from glioblasts? Thanks.

Isnt the definition of glioblast basically pre-glial? 
Taking for fbbt: "Cell that divides to give rise to glial cells." 
Don't see why it should be species specific. 
The axiom in question is in CL atm and I think it should be there, no changes are needed I think.

**Comment by gouttegd** [comment]
> Is it true/known for all taxa that glial cells develop from glioblasts?

I don‘t think we can positively affirm that. Microglia at least certainly don’t seem to derive from glioblasts (see [Ginoux & Prinz, 2015](https://cshperspectives.cshlp.org/content/7/8/a020537.full)).

**Comment by shawntanzk** [comment]
> Microglia at least certainly don’t seem to derive from glioblasts (see [Ginoux & Prinz, 2015](https://cshperspectives.cshlp.org/content/7/8/a020537.full)).

Ah, I think microglial needs a relooking at then.

Currently microglial has the subclass axiom
`'develops from' some 'oligodendrocyte precursor cell'`
which seems wrong 

glial cell therefore also cannot have the axiom `'develops from' some glioblast` - this needs to be pushed down to all child that actually do.

I'll make a ticket specifically for this. If you are experienced enough to do edits on this without much research @gouttegd I would appreciate it, but I do get you're quite busy too, so happy to do it too, just might take a bit of time for me to get to it.
